common/network-shuffle/src/main/java/org/apache/spark/network/shuffle/ExternalShuffleBlockHandler.java

```diff
@@ -66,6 +66,11 @@ public ExternalShuffleBlockHandler(TransportConf conf, File registeredExecutorFi
       new ExternalShuffleBlockResolver(conf, registeredExecutorFile));
   }
 
+  /** ForTesting */
```

Comment:
can you use the same style for labelling the "for testing" methods as already in these files?  I think for all of these java files its `@VisibleForTestin`

Reply:
ok，done